### PR TITLE
Copy changes

### PR DESCRIPTION
**Pull request by fregante**
This felt like an error:

Screen Shot 22

Maybe this new  copy is slightly less about what "**you** in particular" cannot do. There's also `You cannot edit brick: ${id}` somewhere else but maybe "you" in that case is correct.

I wonder if this info is necessary at all? I don't see anything I'd want to edit on the page.

---

Converting to draft to possibly add more copy changes over time.

### CODE REVIEW

**Comment by fregante** [comment]
New change to reduce copy and avoid repetition

## Before

Screen Shot 7

## After

Screen Shot

File: src/devTools/editor/panes/PermissionsPane.tsx [hunk 1]
```diff
@@ -43,17 +43,13 @@ const PermissionsPane: React.FunctionComponent = () => {
...
-          Or, grant temporary access by 1) clicking on the PixieBrix extension
-          in the extensions dropdown, and 2) then refreshing the page.
-        </p>
-      </div>
-      <Button onClick={requestPermissions}>Grant Permanent Access</Button>
+      <p>
+        <Button onClick={requestPermissions}>Grant permanent access</Button>
+      </p>
+      <p>
+        Or grant temporary access by clicking on the PixieBrix extension in the
```
Comment (twschiller) [inline src/devTools/editor/panes/PermissionsPane.tsx:50]: I think we should keep the 1) and 2) in here. It's a long sentence and helps visually indicate there's 2 steps

**Comment by twschiller** [comment]
> Converting to draft to possibly add more copy changes over time

Let's try to avoid having long-lived branches

This looks good, but I have one copy suggestion here: https://github.com/pixiebrix/pixiebrix-extension/pull/630#discussion_r665326783

File: src/devTools/editor/panes/PermissionsPane.tsx (hunk 1 above)
Comment (fregante) [reply src/devTools/editor/panes/PermissionsPane.tsx]: I tried replacing them with a `ol` element but it looked a little worse, so I dropped them altogether. Ideally this should be just one step in the future. I could go look into the browsers’ source code to see what else could trigger activeTab with the best UX.

**Review by twschiller — APPROVED** [review]

File: src/devTools/editor/panes/PermissionsPane.tsx (hunk 1 above)
Comment (twschiller) [reply src/devTools/editor/panes/PermissionsPane.tsx]: The only ones we have right now are context menu and browser action

**Pull request merged**